crawlers/mooncrawl/mooncrawl/metadata_crawler/cli.py

```diff
@@ -38,8 +44,14 @@ def crawl_uri(metadata_uri: str) -> Any:
     result = None
     while retry < 3:
```

Comment:
I sugest to re-write this func to avoid increment of `retry` var only in exceptions. But would do this by default at the end of loop and `if response.status == 200:` then write result and break the `while` loop.
But if this version works well, then probably there are no need any changes.